chapters/statements.tex

```diff
@@ -414,32 +410,27 @@ \subsection{When-Statements}\doublelabel{when-statements}
 when-statement are activated when the scalar or any one of the elements
 of the vector-expression becomes true.
 
-{[}\emph{Example}:
-
-\emph{Algorithms are activated when} \lstinline!x! \emph{becomes \textgreater{} 2:}
-
+\begin{example}
+Algorithms are activated when \lstinline!x! becomes $> 2$:
 \begin{lstlisting}[language=modelica]
 when x > 2 then
   y1 := sin(x);
   y3 := 2*x + y1+y2;
 end when;
 \end{lstlisting}
-\emph{The statements inside the when-statement are activated when
-either} \lstinline!x! \emph{becomes \textgreater{} 2 or} \lstinline!sample(0,2)! \emph{becomes}
-\lstinline!true! \emph{or} \lstinline!x! \emph{becomes less than 5:}
-
+The statements inside the when-statement are activated when either \lstinline!x! becomes $> 2$ or
```

Comment:
I think that equation-fragments such as $> 2$ may be problematic (since > is sort of a binary operator); and it also leads to a bit of font-overuse in one line, and would prefer to keep the previous variant.

Reply:
I changed it back to start with, but isn't this also a sloppy way of writing that should be improved; it is just as much as binary operator with a missing first argument when using `\textgreater{}`.  Wouldn't it be better to say something like:
> are activated on the positive edge of any of the conditions \lstinline!x > 2! or \lstinline!sample(0, 2)!

Reply:
Yes, that would probably make sense.

Reply:
Done, but saying _expressions_ instead of _conditions_ in order to work better with the `sample(2, 0)`.